doc/classes/SpatialMaterial.xml

```diff
@@ -157,10 +157,17 @@
 			Specifies whether to use [code]UV[/code] or [code]UV2[/code] for the detail layer. See [enum DetailUV] for options.
 		</member>
 		<member name="distance_fade_max_distance" type="float" setter="set_distance_fade_max_distance" getter="get_distance_fade_max_distance">
-			Distance at which the object fades fully and is no longer visible.
+            Distance at which the object appears at normal opaqueness.
```

Comment:
It looks like you are using spaces instead of tabs. For proper formatting of the .xml files, you need to use tabs.